x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/action/EnrichShardMultiSearchAction.java

```diff
@@ -259,8 +261,11 @@ protected MultiSearchResponse shardOperation(Request request, ShardId shardId) t
                             }
                             return context.getFieldType(field);
                         });
-                        final SearchHit hit = new SearchHit(scoreDoc.doc, visitor.id());
-                        hit.sourceRef(filterSource(fetchSourceContext, visitor.source()));
+                        final SearchHit hit = new SearchHit(scoreDoc.doc, visitor.id(), breaker);
```

Comment:
Adding accounting here is option and we can do it in a follow-up. 
Added to get more data points from the IT infrastructure.